spec/features/inspection_display_spec.rb

```diff
@@ -13,8 +13,8 @@
     click_on 'Sign in'
   end
   describe "when not logged in" do
-    let(:an_item)  { FactoryGirl.create(:item) }
-    let(:an_inspection) { FactoryGirl.create(:inspection, item: an_item) }
```

Comment:
I'd realized that they were systematically removed in b975d6f2ffb2ff273b94c0f5221bca28747307fd . If that was the standard, I'd erred in putting them in myself.